src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -124,7 +129,13 @@ public BuildScriptSnippet GenerateBashBuildScriptSnippet(BuildScriptGeneratorCon
                 buildProperties[NodeConstants.NodeModulesFileBuildProperty] = compressedNodeModulesFileName;
             }
 
+            var exlcudedDirs = GetDirectoriesToExcludeFromCopyToBuildOutputDir(context);
+
             bool pruneDevDependencies = ShouldPruneDevDependencies(context);
+            bool zipAllOutput = ShouldZipAllOutput(context);
+            // TODO: tightly coupled
```

Comment:
In runtime image the case of the boolean value is important i.e runtime image script expects it to be lower case. Seems like doing a case-INsensitive comparison is not straightforward in bash and also seems to depend on the shell too.